### DISCUSSION TITLE
ConfuserEx 1.5 Release

### DISCUSSION THREAD

**Discussion by mkaring** [mkaring/ConfuserEx#312] [General]
The first release candidate for the upcoming ConfuserEx 1.5 release is tagged, build and ready to be tested.

I would appreciate to get some feedback on this new version and regarding any new issues that appear.

My current to do list for the new release is:

- [x] Update the documentation regarding the added features
- [x] _Get the CLI version on chocolatey white-listed_ - Response of the chocolatey administrators is that the CLI package can't be white-listed until the next release is published.
- [x] Run the new version on the product at my workplace

The release is right now planned for the 11th of February 2021.
This date will be pushed back, only if regressions compared to the 1.4.1 release are reported by then.

**Comment by mkaring** [comment]
Due to a few regressions that were found in the latest release, the new version is pushed by another week to the 18th of February 2021.

The second release candidate is ready: [ConfuserEx 1.5.0 RC2](https://github.com/mkaring/ConfuserEx/releases/tag/v1.5.0-rc2)

**Comment by mkaring** [comment]
Also I'd like to mention that any feedback on how the tests are going, is appreciated.

**Comment by KvanTTT** [comment]
What about merging of [renaming improvements](https://github.com/mkaring/ConfuserEx/pull/305)? It improves the quality of obfuscation.

**Reply by mkaring** [reply]
There are changes in that pull-request that likely have unforeseen side effects. If you limit the changes this pull request to the things that are actually required for the linked issues, I'll look into it.
As this pull request is right now, I'll merge it for the next release, after I had much more time to test what those changes do.

**Reply by mkaring** [reply]
I'm looking at another release late April or early May. We'll get your changes in for that release.

**Comment by mkaring** [comment]
ConfuserEx 1.5 has been released.
